### ISSUE TITLE
-Wmissing-braces in drivers/char/hw_random/optee-rng.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#369]
```
drivers/char/hw_random/optee-rng.c:80:31: warning: suggest braces around initialization of subobject [-Wmissing-braces]
        struct tee_param param[4] = {0};
                                     ^
                                     {}
drivers/char/hw_random/optee-rng.c:177:31: warning: suggest braces around initialization of subobject [-Wmissing-braces]
        struct tee_param param[4] = {0};
                                     ^
                                     {}
drivers/char/hw_random/optee-rng.c:212:48: warning: suggest braces around initialization of subobject [-Wmissing-braces]
        struct tee_ioctl_open_session_arg sess_arg = {0};
                                                      ^
                                                      {}
3 warnings generated.
```

Patch sent: https://lore.kernel.org/lkml/20190219032240.22935-1-natechancellor@gmail.com/

**Comment by nickdesaulniers** [comment]
accepted: https://git.kernel.org/pub/scm/linux/kernel/git/arm/arm-soc.git/commit/?h=arm/drivers&id=56410c0cb43e26a94af094aceab45a5ea38a1a38

**Comment by nathanchance** [comment]
Fixed in `next-20190221`.
